abc.templates/infra/spec.yaml

```diff
@@ -60,7 +58,7 @@ steps:
         - to_replace: 'REPLACE_BUCKET_NAME'
           with: '{{.terraform_state_bucket}}'
         - to_replace: 'REPLACE_BUCKET_PREFIX'
-          with: '{{.terraform_state_prefix}}'
+          with: '{{.subdirectory}}'
```

Comment:
Should we use dest instead? What if someone uses dest and this flag?

Reply:
Lets merge this for now and re-work if needed

Reply:
Will address this in following changes